pom.xml

```diff
@@ -739,6 +739,127 @@
+            <artifactId>httpclient</artifactId>
+            <version>4.5.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpcore</artifactId>
+            <version>4.4.6</version>
+        </dependency>
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-beans</artifactId>
+            <version>2.5.6.SEC03</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.9.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-embedder</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.sisu</groupId>
+            <artifactId>org.eclipse.sisu.plexus</artifactId>
+            <version>0.3.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-aether-provider</artifactId>
+            <version>3.3.1</version>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.icon-shim</groupId>
+            <artifactId>icon-set</artifactId>
+            <version>2.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.codehaus.plexus</groupId>
+            <artifactId>plexus-classworlds</artifactId>
+            <version>2.5.2</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-compat</artifactId>
+            <version>3.5.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-tree</artifactId>
+            <version>5.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci</groupId>
+            <artifactId>symbol-annotation</artifactId>
+            <version>1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>com.infradna.tool</groupId>
+            <artifactId>bridge-method-annotation</artifactId>
```

Comment:
Part of core, should not be here.

Reply:
Thats right but core has older version than the one from github-api plugin. How do we do it?

```
Require upper bound dependencies error for com.infradna.tool:bridge-method-annotation:1.13 paths to dependency are:
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-org.jenkins-ci.main:jenkins-core:2.46.3
    +-com.infradna.tool:bridge-method-annotation:1.13
and
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-io.jenkins.blueocean:blueocean-pipeline-api-impl:1.3.0-beta-3-SNAPSHOT
    +-org.jenkins-ci.plugins:git:3.5.1
      +-com.infradna.tool:bridge-method-annotation:1.13
and
+-io.jenkins.blueocean:blueocean-bitbucket-pipeline:1.3.0-beta-3-SNAPSHOT
  +-io.jenkins.blueocean:blueocean-pipeline-api-impl:1.3.0-beta-3-SNAPSHOT
    +-org.jenkins-ci.plugins:github-branch-source:2.2.3
      +-org.jenkins-ci.plugins:github-api:1.86
        +-org.kohsuke:github-api:1.86
          +-com.infradna.tool:bridge-method-annotation:1.14
```

Reply:
This is a bug in `github-api-plugin/pom.xml`. I will file a PR to fix it. In the short term you can exclude this dep, so as to pick it up from core.

Reply:
https://github.com/jenkinsci/github-api-plugin/pull/15